biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1637,13 +1637,17 @@ subroutine fire_litter_fluxes(currentSite, currentPatch, &
              ! across ground surface. 
              ! -----------------------------------------------------------------------
              
-             sapw_m   = currentCohort%prt%GetState(sapw_organ, element_id)
-             struct_m = currentCohort%prt%GetState(struct_organ, element_id)
-             leaf_m   = currentCohort%prt%GetState(leaf_organ, element_id)
-             fnrt_m   = currentCohort%prt%GetState(fnrt_organ, element_id)
-             store_m  = currentCohort%prt%GetState(store_organ, element_id)
-             repro_m  = currentCohort%prt%GetState(repro_organ, element_id)
-             
+			if ( prt_params%woody(pft) == itrue) then !trees only, May require additional accounting for the atm_fluxes when PFT is grass. 
```

Comment:
@ZacharyRobbins there is something going on with spacing here. Can you fix this so that the if statement is moved back to the left?

Reply:
I believe I have resolved the spacing issue, and return fine roots to their original location.

Reply:
Is this outstanding still?